tests/test_dodecaSkyMap.py

```diff
@@ -57,6 +57,70 @@ def testSha1Compare(self):
         skyMap = self.getSkyMap(config=config)
         self.assertNotEqual(skyMap, defaultSkyMap)
 
+    def testTractInfoGetPolygon(self):
+        shiftAngle = 0.01*afwGeom.arcseconds
+        skyMap = DodecaSkyMap()
+        for tractInfo in skyMap:
+            innerPolygon = tractInfo.getInnerSkyPolygon()
+            centerCoord = tractInfo.getCtrCoord()
+            for vertex in tractInfo.getVertexList():
+                # offset vertex a bit towards the center to be sure it will be in the polygon
+                # avoiding roundoff errors
+                bearingToCenter = vertex.bearingTo(centerCoord)
+                vertexShiftedIn = vertex.offset(bearing=bearingToCenter, amount=shiftAngle)
+                self.assertTrue(innerPolygon.contains(vertexShiftedIn.getVector()))
+
+                vertexShiftedOut = vertex.offset(bearing=bearingToCenter, amount=-shiftAngle)
+                self.assertFalse(innerPolygon.contains(vertexShiftedOut.getVector()))
+
+            outerPolygon = tractInfo.getOuterSkyPolygon()
+            pixelCorners = afwGeom.Box2D(tractInfo.getBBox()).getCorners()
+            wcs = tractInfo.getWcs()
+            skyCorners = wcs.pixelToSky(pixelCorners)
+            for skyCorner in skyCorners:
+                bearingToCenter = skyCorner.bearingTo(centerCoord)
+                cornerShiftedIn = skyCorner.offset(bearing=bearingToCenter, amount=shiftAngle)
+                cornerShiftedOut = skyCorner.offset(bearing=bearingToCenter, amount=-shiftAngle)
+                self.assertTrue(outerPolygon.contains(cornerShiftedIn.getVector()))
+                self.assertFalse(outerPolygon.contains(cornerShiftedOut.getVector()))
```

Comment:
This is the same code as above for the inner polygon. Can you factor it into a function?